packages/plugin-core/src/commands/pods/GoogleDocsExportPodCommand.ts

```diff
@@ -0,0 +1,176 @@
+          id: opts.connectionId,
+        });
+
+        if (!config) {
+          window.showErrorMessage(
+            `Couldn't find service config with the passed in connection ID ${opts.connectionId}.`
+          );
+          return;
+        }
+        accessToken = config.accessToken;
+        refreshToken = config.refreshToken;
+        expirationTime = config.expirationTime;
+        connectionId = config.connectionId;
+      } else {
+        // Prompt User to pick an gdoc connection, or create a new one
+        // (which will stop execution of current pod command)
+        const connection =
+          await PodUIControls.promptForExternalServiceConnectionOrNew<GoogleDocsConnection>(
+            ExternalService.GoogleDocs
+          );
+
+        if (!connection) {
+          return;
+        }
+
+        connectionId = connection.connectionId;
+        accessToken = connection.accessToken;
+        refreshToken = connection.refreshToken;
+        expirationTime = connection.expirationTime;
+      }
+    }
+
+    // Get the export scope
+    const exportScope =
+      opts && opts.exportScope
+        ? opts.exportScope
+        : await PodUIControls.promptForExportScope();
+
+    if (!exportScope) {
+      return;
+    }
+
+    const inputs = {
+      exportScope,
+      accessToken,
+      refreshToken,
+      ...opts,
+      podType: PodV2Types.GoogleDocsExportV2,
+      expirationTime,
+      connectionId,
+    };
+    if (!isRunnableGoogleDocsV2PodConfig(inputs)) {
+      let id = inputs?.podId;
+      if (!id) {
+        const picked = await PodUIControls.promptForGenericId();
+
+        if (!picked) {
+          return;
+        }
+        id = picked;
+      }
+      const configPath = ConfigFileUtils.genConfigFileV2({
+        fPath: path.join(getExtension().podsDir, "custom", `config.${id}.yml`),
+        configSchema: GoogleDocsExportPodV2.config(),
+        setProperties: _.merge(inputs, {
+          podId: id,
+          connectionId: inputs.connectionId,
+        }),
+      });
+      await VSCodeUtils.openFileInEditor(vscode.Uri.file(configPath));
+      //TODO: Modify message:
+      return;
+    } else {
+      return inputs;
+    }
+  }
+
+  async onExportComplete(opts: {
+    exportReturnValue: GoogleDocsExportReturnType;
+    payload: NoteProps;
+    config: RunnableGoogleDocsV2PodConfig;
+  }) {
+    const { exportReturnValue, payload } = opts;
+    if (ResponseUtil.hasError(exportReturnValue)) {
+      this.L.error(exportReturnValue.error);
+      window.showErrorMessage(
+        "Error while running Google Docs Export Pod: " + exportReturnValue
+      );
+      return;
+    }
+    if (exportReturnValue.data?.documentId) {
+      await this.updateNoteWithDocumentId({
+        documentId: exportReturnValue.data?.documentId,
+        note: payload,
+      });
+    }
+    vscode.window.showInformationMessage(
+      "Finished running Google Docs export pod."
```

Comment:
nit: can we expand this information message to give an indication of the gdoc file name that was exported to?